### ISSUE TITLE
IndexSizeError: Index or size is negative or greater than the allowed amount

### ISSUE THREAD

**Issue by jmurn** [osTicket/osTicket#4733]
### Description

when you try paste image from clipboard to ticket response then nothing happened.

In browser console this error appear:
IndexSizeError: Index or size is negative or greater than the allowed amount redactor.min.js:1
nodeToCaretPositionFromPoint https://****/js/redactor.min.js?d4e240b line 12 > eval:1 nodeToCaretPositionFromPoint self-hosted:976 insert https://****/js/redactor.min.js?d4e240b line 12 > eval:1 insert self-hosted:978 onreadystatechange https://****/js/redactor.min.js?d4e240b line 12 > eval:1 i jQuery

Issue is only in Mozilla firefox. Google chrome works fine.

### Versions

osTicket Version | v1.11 (d4e240b)     —  Up to date
-- | --
Web Server Software | Apache/2
MySQL Version | 10.3.12
PHP Version | 7.0.33

**Comment by ntozier** [comment]
I do not believe that this is an osTicket issue.  The editor is a 3rd party editor so if it works for one browser and not the other I would think that this would want to be raised with the editor support.
